### ISSUE TITLE
git-secret-changes use of echo adds a newline

### ISSUE THREAD

**Issue by BrandonCsSanders** [sobolevn/git-secret#291]
What are the steps to reproduce this issue?
-------------------------------------------
1. don't change FILE.pem.secret
2. run git-secret changes FILE.pem.secret

What happens?
-------------
`git-secret changes` shows that a newline at the end of the file was removed
```
--- /dev/fd/63	2018-12-28 08:43:13.000000000 -0800
+++ FILE.pem	2018-12-27 17:37:37.000000000 -0800
@@ -20,4 +20,4 @@
ASDFASDFASDFADFSADFSAFSDASDF...elided...
+-----END RSA PRIVATE KEY-----
\ No newline at end of file
```

What were you expecting to happen?
----------------------------------
no changes

Any logs, error output, etc?
----------------------------

```
    diff_result=$(diff -u <(echo "$decrypted") "$path") || true
```

Should instead be

```
    diff_result=$(diff -u <(echo -n "$decrypted") "$path") || true
```

What versions of software are you using?
----------------------------------------
0.2.4

**Comment by sobolevn** [comment]
@BrandonCsSanders thanks for the bug report! Can you please send a PR with the fix?

**Comment by joshrabinowitz** [comment]
Fixed in #293
